google/cloud/aiplatform/models.py

```diff
@@ -324,6 +324,45 @@ def _create(
             credentials=credentials,
         )
 
+    def _construct_sdk_resource_from_gapic(
+        self,
```

Comment:
It seem like this should be converted to a classmethod along with the implementation in the base class.